gym/lib/gym/error_handler.rb

```diff
@@ -54,6 +54,7 @@ def handle_build_error(output)
         print_xcode_path_instructions
         print_xcode_version
         print_full_log_path
+        print_environment_information
```

Comment:
Any thoughts on making some of these print methods only happen if the user is running _fastlane_ in verbose mode?